Antidote/ActiveCallViewController.m

```diff
@@ -156,6 +209,30 @@ - (void)installConstraints
         make.centerY.equalTo(self.containerView);
     }];
 
+    [self.callMenuButton makeConstraints:^(MASConstraintMaker *make) {
+        make.width.equalTo(kButtonSide);
+        make.height.equalTo(kButtonSide);
+        make.right.equalTo(self.topViewContainer.rightMargin);
+        make.top.equalTo(self.topViewContainer.topMargin);
+    }];
+
+    [self.tableViewOfPausedCalls makeConstraints:^(MASConstraintMaker *make) {
+        make.top.equalTo(self.topViewContainer.bottom);
+        make.width.equalTo(self.view);
+        make.height.equalTo(self.view.bounds.size.height / 3.0);
```

Comment:
I'm not sure it this will work after you rotate device. I think it is better instead to set `bottom` constraint to `(self.view.bottom).offset(-kSomeConstant);`